docs/30_spectra/10_organic_chemistry/chromatography/introduction/README.md

```diff
@@ -0,0 +1,11 @@
+# GC/LC-MS chromatography 
+
+https://en.wikipedia.org/wiki/Gas_chromatography
+
+https://en.wikipedia.org/wiki/Liquid_chromatography%E2%80%93mass_spectrometry
+
+Chromatography is a laboratory technique allowing to separate mixtures. A sample is dissolved in a fluid (gas or liquid) which travels through a solid system (column, plate, sheet, capillary tube). The fluid is often called the mobile phase and the system is termed the stationary phase. Since different constituents of the mixture have different affinities for the stationary and the mobile phases, they will move with different apparent velocities. This difference in the apparent velocities allows to separate the components.
+
+The name of the specific type of chromatography depends on the physical state of the mobile phase, GC for gas and LC for liquid. These techniques are often combined with mass spectrometry. First, a mixture is separated and then the components are identified.
```

Comment:
maybe mention that the "components are identified using MS" 

and, personally, I wouldn't write "identified" as, in general, you only measure the spectrum and might not identify anything